src/main/java/duke/Storage.java

```diff
@@ -29,8 +31,8 @@ public Storage(String pathname) {
      * @param taskList A TaskList with no tasks.
      * @throws IOException If file is not found or error reading file.
      */
```

Comment:
It would be good to declare that the MissingInfoException is being thrown in the javaDoc for the load method.